### DISCUSSION TITLE
How to loop a region

### DISCUSSION THREAD

**Discussion by logesh2496** [katspaugh/wavesurfer.js#2948] [Q&A]
I have multiple regions created, need to way to enable looping. Any solution either setting loop to true during the addRegion or during regions[id].playLoop() should be fine. I'm on v6.2.0. 

Thanks all

**Comment by katspaugh** [comment]
For v6, refer to these docs for Regions: https://wavesurfer-js.org/plugins/regions

**Comment by logesh2496** [comment]
worked in v6.6.4, thanks
